src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -102,10 +108,112 @@ def validate_role(self, role):
         return role
 
 
+class OrganizationMemberIndexQueryParamSerializer(serializers.Serializer):
+    email = serializers.CharField(
+        required=False,
+        default=None,
+        help_text="an email address to filter by.",
+    )
+    scope = serializers.ChoiceField(
+        required=False,
+        choices=settings.SENTRY_SCOPES,
+        help_text="filter on members with a specific scope.",
+    )
+
+    role = serializers.ChoiceField(
+        choices=roles.get_choices(),
+        required=False,
+        help_text="filter on members with a role.",
+    )
+    isInvited = serializers.BooleanField(
+        required=False, help_text="filter on if the member is in an invited state."
+    )
+    ssoLinked = serializers.BooleanField(
+        required=False, help_text="filter on members who have linked an SSO identity."
+    )
+    has2fa = serializers.BooleanField(
+        required=False, help_text="filter on members who have 2fa enabled."
+    )
+    hasExternalUsers = serializers.BooleanField(
+        required=False, help_text="filter on members who have an external user linked."
+    )
+    query = serializers.CharField(
+        required=False,
+        help_text="filter by an email address, but look at secondary email's and primary account emails as well.",
+    )
+    expand = serializers.MultipleChoiceField(
+        choices=["externalUsers"],
+        required=False,
+        help_text="whether or not to expand the member's external users",
+    )
+
+
+@public(methods={"GET"})
+@extend_schema(tags=["Members"])
 class OrganizationMemberIndexEndpoint(OrganizationEndpoint):
     permission_classes = (MemberPermission,)
 
+    @extend_schema(
+        operation_id="Query an Organization's Members",
+        parameters=[GLOBAL_PARAMS.ORG_SLUG, OrganizationMemberIndexQueryParamSerializer],
+        request=None,
+        responses={
+            200: inline_sentry_response_serializer(
+                "OrganizationMembersList",
+                List[organization_member_serializers.OrganizationMemberResponse],
+            )
+        },
+        examples=[
+            OpenApiExample(
```

Comment:
i'm wondering if folks would prefer if examples were defined below the endpoint at the end of the file, or in a separate file entirely. I am used to using code folding etc. but I'm curious if others find this annoying.

Reply:
Some endpoints will have pretty beefy examples. We might want to explore having examples in a separate file that is read in.

Reply:
I've put the example in a separate file -- within a new directory created in `/api/` called examples, with each endpoint getting their own file for examples there.